### DISCUSSION TITLE
2022 roadmap -  Reolinkers’ Week - let's speak common voice

### DISCUSSION THREAD

**Discussion by macxq** [fwestenberg/reolink_dev#432] [Show and tell]
During last week _**Reolinkers’ Week**_  the 2022 roadmap has been shared - it is chance for as to speak the same voice.

It the the https://community.reolink.com/topic/2832/reolinkers-week-day-3-product-feature-roadmap/4?page=1 user **tenback** ask for

> Custom HTTP(s) query, where you can enter an url, when certain motion is detected there will be a http get on this url, so extern systems can be triggered without adding too much overhead.

How I understand how this plugin works - it would be also beneficial for us :) I'm encouraged you to like this request!

**Comment by cpainchaud** [comment]
It's called a webhook or callback url. And we also need them to support it with HTTPS
